store/sqlstore/user_store.go

```diff
@@ -1311,6 +1311,10 @@ func (us SqlUserStore) Count(options model.UserCountOptions) (int64, error) {
 		query = query.Where("u.DeleteAt = 0")
 	}
 
+	if !options.IncludeRemoteUsers {
+		query = query.Where("u.RemoteId = '' || u.RemoteId IS NULL")
```

Comment:
Is this correct? I don't think `||` works. You'll need to use `OR`. Let's also add unit tests to avoid any bugs like this.

Reply:
:facepalm: I was about to add a unit test for checking all this. Sorry for early adding you to the review.

Reply:
Added the tests, and fixing the store code